src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -2960,29 +2929,6 @@ public static void InvokeQuotedLambda(bool useInterpreter)
             Assert.Equal(6, d());
         }
 
-        [Theory(Skip = "870811")]
```

Comment:
I'll do that, though I'm wondering if the autoquote and/or nested compile tests could have a similar issue, though they don't close over anything, so if that's where the issue is, they should be fine.